src/content/docs/Getting Started/projects.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Projects
+description: Learn to create C3 projects
+sidebar:
+  order: 31
+---
+import { FileTree } from '@astrojs/starlight/components';
+
+:::note[Not installed the C3 compiler yet?]
+[Download C3](/install-c3/prebuilt-binaries/), available on Mac, Windows and Linux.
+:::
+
+## Projects in C3
+Projects are *optional*, but are a good way to manage compiling code when there
+ are a lot of files and modules.
+They also allow you so specify libraries to link and define how your project should be built for specific targets.
+
+### 💡 Creating a new project
+
+The `c3c init` command will create a new directory containing your project structure.
+It requires a name of the project, we will use `myc3project` in its place.
+
+```bash
+c3c init myc3project
+```
+
+You can also customize the path where the project will be created or specify a *template*. For more information check the [init command reference](/build-your-project/build-commands/#init-a-new-project).
+
+### 📁 Project structure
+
+If you check the directory that was created you might find it a bit confusing with a bunch of different directories, but worry not because if you expand them you will realise that most of them are actually empty!
+
+<FileTree>
+- build/
+- docs/
+- lib/
+- resources/
+- scripts/
+- src/
+  - main.c3
+- test/
+- LICENSE
+- project.json
+- README.md
+</FileTree>
+
+Let's go over some of the useful ones;
+- `./build` is where your temporary files and build results will end up
+- `./docs` are the overall documentations for your code
+- `./lib` is where C3 libraries (with the .c3l suffix) end up
+- `./resources` are for any resources like images, sound effects etc.
+- `./scripts` is for various scripts, including .c3 scripts that can run at compile time to generate code
+- `./src` is what we care about as it's where a majority of our code will go and it now contains `main.c3` with "Hello World" example
+- `project.json` file is where you add more data about your project, sort of like package.json for node.js
+- `LICENSE` file for your code license
+- `README.md` file where you can describe your project
+
+### 🔧 Building the project
+
+Assuming you have successfully initialized a project as seen above, we can now look at how to compile it.
+
+We can simply build it with:
```

Comment:
Let's move the c3c run part so it's first before `c3c build`, so they can see the most important information first